src/pushsource/_impl/model/ami.pyi

```diff
@@ -0,0 +1,32 @@
+from collections import Sequence
+from typing import Text, Optional
```

Comment:
You're right. I started with the classes in `model/` and since then talked myself out of using `Text` so we're not forcing people to worry about `unicode` in Python 2 instead of plain strings. I have a note to myself to go back to the classes in `model/` and strip out uses of `Text` as well as add stubs for dunders, and I have just started on that recently.